keyboards/lpad/plus/readme.md

```diff
@@ -0,0 +1,15 @@
+# lpadplus
+
+![lpadplus](https://github.com/bululau/lpad/blob/main/plus/IMG_0844.JPG)
+
+lpad plus
+
+* Keyboard Designer: [bululau](https://github.com/bululau)
+* Keyboard Maintainer: [bululau](https://github.com/bululau)
+* Hardware Availability: https://github.com/bululau/lpad/tree/main/plus
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```